src/rows.ts

```diff
@@ -83,7 +95,17 @@ export class Rows {
         throw this._db._error(status);
         break;
     }
-    return { value: row, done: false };
+
+    if (this._options?.asObjects) {
```

Comment:
Yeah, I think you might have left this in from the previous commit. I think this should live exclusively inside the `RowObjects`